Fitness/Controllers/GymDetailViewController.swift

```diff
@@ -342,15 +376,21 @@ class GymDetailViewController: UIViewController {
 
         let facilitiesHeight = facilitiesData.count*20
         let todaysClassesHeight = classesTableView.numberOfRows(inSection: 0)*112
+        let height: Int
 
-        //THIS MUST BE CHANGED IF ANY OF THE SCREEN'S HARD-CODED HEIGHTS ARE ALTERED
-        let height = 427 + dropHoursHeight + 282 + facilitiesHeight + 137 + todaysClassesHeight
+        // THIS MUST BE CHANGED IF ANY OF THE SCREEN'S HARD-CODED HEIGHTS ARE ALTERED
+        if gym.isOpen {
+            height = 427 + dropHoursHeight + 282 + facilitiesHeight + 137 + todaysClassesHeight
+        } else {
+            height = 427 + dropHoursHeight + 89 + facilitiesHeight + 137 + todaysClassesHeight
+        }
+        
         scrollView.contentSize = CGSize(width: view.frame.width, height: CGFloat(height))
```

Comment:
you can make a scrollView work with AutoLayout. I'd love to show you how on Wednesday!